vue/sbc-common-components/src/services/launchdarkly.services.ts

```diff
@@ -0,0 +1,61 @@
+import { initialize, LDFlagSet } from 'launchdarkly-js-client-sdk'
```

Comment:
This looks OK, but I'm wondering why it doesn't look more like this https://github.com/bcgov/business-filings-ui/blob/master/src/common/FeatureFlags.ts ?

Would you recommend that Entities team update their implementation to more closely match yours?

Reply:
It wont be necessary, I have done it this way since we will be using the session storage to save the flags. so the initialization and all done inside the class itself, only if the flags are not available in the session. which is for avoiding the unnecessary calls to LD.